### DISCUSSION TITLE
Is Nugget Doom supposed to be compatible with the Legacy of Rust resources?

### DISCUSSION THREAD

**Discussion by klearscreen** [MrAlaux/Nugget-Doom#126] [Q&A] [answered]
Legacy of Rust offers its resources Id1-res.wad and Id1-weap.wad to replace the plasma gun and BFG with the new weapons from its expansion.
According to the doomwiki they follow the MBF21 standard, so they should be compatible with Nugget Doom, but loading them causes hall of mirrors effects, easily seen on TNT map 30:
![TNTmap30](https://github.com/user-attachments/assets/2a7adc67-19fa-45fe-a72a-6b8aae3aefba)

**Comment by MrAlaux** [comment, marked as answer]
`id1-res.wad` includes a `TEXTURE1` lump that seemingly conflicts with TNT's. If you're not into Doom modding, that lump is basically the list of textures, of which there can only be one loaded at a time. In other words, this actually seems to be an incompatibility between LoR's resources and TNT.

The current Nugget release is mostly compatible with LoR's resources, save for some cosmetic stuff involving skies and intermission screens, but those aren't critical. Even then, the next release should include support for those features.
